acceptance/pe/tests/basic.rb

```diff
@@ -0,0 +1,20 @@
+require 'scooter'
+
+test_name "basic validation of puppetdb-cli subcommands" do
+  puppet_query_on(client, "--help")
+  puppet_db_on(client, "--help")
+
+  step 'setup user and get token with puppet-access' do
+    console_dispatcher = Scooter::HttpDispatchers::ConsoleDispatcher.new(master)
+    administrator_role = console_dispatcher.get_role_by_name('Administrators')
+    user = console_dispatcher.generate_local_user
+    console_dispatcher.add_user_to_role(user, administrator_role)
+    login_with_puppet_access_on(client, user)
+  end
+
+  puppet_query_on(client, "'nodes{}'")
```

Comment:
Interesting. With a more recent build, the tool expects a 'nodes' command rather than a query string. So, this command now fails when quoted. The single quotes in this command now need to be removed in order to get this test to pass.